### PR TITLE
Globally fix slow mousewheeling issues, with scaling!

### PR DESCRIPTION
**Pull request by WeaverThree**
`JScrollPane`. They're everywhere. Sometimes the mousewheel scrolling works fine with them, sometimes it doesn't. There doesn't seem to be any pattern to when it works.

MegaMek and MegaMekLab already control their `JScrollPane`s with `get*ScrollBar().setUnitIncrement(int)`, seemingly doing it for each one separately. HQ only does it for the StratCon map.

Something like two-thirds of HQ's are anonymous, which makes setting properties on them a bother.

This implements a drop-in replacement for `JScrollPane`, `JScrollPaneWithSpeed`, which sets its own increment modified by the GUI scale factor, and replaces all uses of `JScrollPane` with it. (Except that Stratcon Map one.) This provides a single point for tuning and would allow for customization down the line.

### CODE REVIEW

File: MekHQ/src/mekhq/gui/utilities/JScrollPaneWithSpeed.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2019-2024 - The MegaMek Team. All Rights Reserved.
```
Comment (IllianiCBT) [inline MekHQ/src/mekhq/gui/utilities/JScrollPaneWithSpeed.java:2]: New files only need the current year

File: MekHQ/src/mekhq/gui/utilities/JScrollPaneWithSpeed.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+/**
+ * It's a JScrollPane that manages its scrollspeed based on the UI scale
+ */
+public class JScrollPaneWithSpeed extends JScrollPane {
+    final static int BASE_INCREMENT = 16;
+
+    /**
+     * @see JPanel#JPanel()
+     */
+    public JScrollPaneWithSpeed() {
```
Comment (IllianiCBT) [inline MekHQ/src/mekhq/gui/utilities/JScrollPaneWithSpeed.java:36]: Would it make sense to use overrides in this class?

Comment (WeaverThree) [reply MekHQ/src/mekhq/gui/utilities/JScrollPaneWithSpeed.java:36]: You mean `@Override`?  I didn't know that worked on constructors

Comment (IllianiCBT) [reply MekHQ/src/mekhq/gui/utilities/JScrollPaneWithSpeed.java:36]: Sorry, that was just poor placement on my part. I had to stick the comment somewhere, but it was more a general comment on the class as a whole.

Comment (IllianiCBT) [reply MekHQ/src/mekhq/gui/utilities/JScrollPaneWithSpeed.java:36]: Conversation can be ignored.

**Review by IllianiCBT — APPROVED** [review]
LGTM

**Pull request merged**